tests/MongoDB.Driver.Tests/Specifications/transactions/TransactionTestRunner.cs

```diff
@@ -37,11 +39,12 @@
 
 namespace MongoDB.Driver.Tests.Specifications.transactions
 {
-    public sealed class TransactionTestRunner
+    public sealed class TransactionTestRunner : ITestRunner
```

Comment:
Interface needed for new `testRunner` target in YML tests and so that the testRunner can own the failPoints for disposal

Reply:
Good idea.

Reply:
For a moment I thought there was a bug here because it looked like TransactionTestRunner was not IDisposable.

I think we should move the IDisposable from ITestRunner to here.

We actually don't want other classes that have been passed an instance of _testRunner to be able to call Dispose, so it really is more correct for this class to be the one that is IDisposable.